static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -116,6 +140,19 @@ const AuditLogList = ({
   );
 };
 
+const SentryAvatar = styled(ActivityAvatar)`
+  margin-right: ${space(1)};
+`;
+
+const Name = styled('div')`
+  font-weight: 600;
+  font-size: 15px;
+`;
```

Comment:
```suggestion
const Name = styled('strong')`
  font-size: ${p => p.theme.fontSizeMedium};
`;
```

Using `strong` as your tag will get you bolding for free, and we should try to use font sizes defined in the theme configuration.